frontend/components/commandPalette/CommandPaletteItem.vue

```diff
@@ -0,0 +1,94 @@
+<!-- TODO: needs to handle focus / tab-to events, get rid of focus ring -->
```

Comment:
Sub directory here should be `command-palette` :)

Reply:
I'm also going to move this to modal as the other ones there have their sub components together (i.e. QR code, etc). Will only be used in the modal, so it's ok to keep it there :)

Reply:
Thanks for splitting out the functionality!